admin/app/views/workarea/admin/catalog_variants/index.html.haml

```diff
@@ -34,6 +34,7 @@
             %th= t('workarea.admin.fields.fulfillment')
             %th.align-center= t('workarea.admin.fields.inventory')
             %th.align-right= t('workarea.admin.fields.pricing')
+            %th.align-right= t('workarea.admin.fields.shipping')
```

Comment:
Right now, this just says "Shipping" to match the other column names like "Fulfillment", "Inventory", etc. But when you read it on the table, it doesn't really make much sense. In my opinion, a more descriptive name for this would be "Packaging", because the attributes expressed in a `Shipping::Sku` describe the weight and dimensions of a given SKU.